src/page/shipping-step.page.ts

```diff
@@ -6,7 +6,7 @@ export class ShippingStepPage {
 
   constructor () {
     this.acceptTermsAndConditionCheck = $('#cgv');
-    this.proceedToCheckoutButton = $('#form > p > button > span');
+    this.proceedToCheckoutButton = $('.cart_navigation > .button > span');
```

Comment:
try to find a better locator, something that uses the name or title attribute for example, in that way, we can understand what element you are referring